Assets/GothicVR/Scripts/Bootstrap/PhoenixBootstrapper.cs

```diff
@@ -1,31 +1,33 @@
-using System;
 using System.Diagnostics;
 using System.IO;
 using AOT;
 using GVR.Creator;
 using GVR.Debugging;
-using GVR.Demo;
 using GVR.Manager;
 using GVR.Manager.Settings;
 using GVR.Phoenix.Interface;
 using GVR.Phoenix.Interface.Vm;
 using GVR.Util;
 using PxCs.Helper;
 using PxCs.Interface;
-using TMPro;
-using Unity.VisualScripting;
-using UnityEngine.TextCore.LowLevel;
+using UnityEngine.XR;
+using UnityEngine;
 using Debug = UnityEngine.Debug;
 
 namespace GVR.Bootstrap
 {
     public class PhoenixBootstrapper : SingletonBehaviour<PhoenixBootstrapper>
     {
         private bool _loaded = false;
+        public GameObject configurationMessage;
+        public GameObject filePickerButton;
 
         private void Start()
         {
             PxLogging.pxLoggerSet(PxLoggerCallback);
+
+            // Just in case we forgot to disable it in scene view. ;-)
+            configurationMessage.SetActive(false);
```

Comment:
I think we also should disable the button, right? Just in case we forgot to disable it inside the scene.

Reply:
```suggestion
            configurationMessage.SetActive(false);
            filePickerButton.SetActive(false);
```